components/match2/wikis/apexlegends/get_match_group_copy_paste_wiki.lua

```diff
@@ -35,7 +35,7 @@ function WikiCopyPaste.getMatchCode(bestof, mode, index, opponents, args)
 		'|p_kill=1 |p1=12 |p2=9 |p3=7 |p4=5 |p5=4 |p6=3 |p7=3 |p8=2 |p9=2 |p10=2 |p11=1 |p12=1 |p13=1 |p14=1 |p15=1',
 		{INDENT .. '|twitch=|youtube='},
 		Array.map(Array.range(1, bestof), function(mapIndex)
-			return INDENT .. '|map' .. mapIndex .. '={{Map|date=|finished=|map=|vod=|stats=}}'
+			return INDENT .. '|map' .. mapIndex .. '={{Map|date=|finished=|map=|vod=}}'
```

Comment:
intended to remove stats here?

Reply:
Yes, no display of it in match summary, so removed it from copy paste and input parsing